### PR TITLE
Feature/improve error generation

### PR DESCRIPTION
**Pull request by Nainterceptor**

### CODE REVIEW

**Comment by Nainterceptor** [comment]
[Retravailler sur les erreurs émises par l'API](https://trello.com/c/stMzftZn/5-retravailler-sur-les-erreurs-%C3%A9mises-par-lapi)

File: tests/routes/campuses.test.mjs [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import chai from 'chai';
+import request, { generateUserJWTHeader } from '../request';
+import Campus, { generateDummyCampus } from '../models/campus';
+
+const { expect } = chai;
+
+describe('Test the campus route', () => {
+  it('Same ID should be saved once', async () => {
```
Comment (Rodrive) [inline tests/routes/campuses.test.mjs]: This message is not like the others (e.g "'POST API endpoint should create a new user'" or "POST API endpoint should create a new role")

File: routes/rides.mjs [hunk 2]
```diff
@@ -14,10 +15,6 @@ router.post(
   maskOutput,
   async (ctx) => {
     const { request: { body } } = ctx;
-
-    if (await Ride.findById(body.id)) {
-      throw new Error('Ride already exists.');
```
[the remark this answers is not in the document]
Comment (Nainterceptor) [reply routes/rides.mjs]: As said in the commit comment, id is autogenerated for a ride, so this test was probably a wront copy/paste

File: tests/routes/jwt.test.mjs [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        expect(
+          jwt.decode(token, { secret: config.get('token:secret') }),
+        )
+          .to.deep.include({ email: user.email, id: user.id });
+      } finally {
+        await user.remove();
+      }
+    });
+
+    it('It should return a valid token on generate', async () => {
```
[the remark this answers is not in the document]
Comment (Nainterceptor) [reply tests/routes/jwt.test.mjs]: Wtf, removed, thanks

**Pull request merged**